doors_dashboards/components/scattermap.py

```diff
@@ -264,31 +268,36 @@ def update_selected_dropdown_point_on_scattermap(general_data, current_figure):
                         group_value = (
                             group_values[0] if len(group_values) > 1 else group_values
                         )
-                    geometry = df[df[label] == group_value]["geometry"]
-                    lon, lat = geometry.x.values, geometry.y.values
-                else:
-                    lon, lat = None, None
+                geometry = df[df[label] == group_value]["geometry"]
+                geometries = np.array([geometry.x.values, geometry.y.values])
+                geometries = np.unique(geometries, axis=1)
+                lon = list(geometries[0])
+                lat = list(geometries[1])
+                text = [group_value] * len(lon)
+                general_data["selected_data"] = {"lon": lon, "lat": lat, "label": text}
 
             if lon is None or lat is None:
                 return no_update
 
             highlighted_trace = go.Scattermapbox(
                 lat=lat,
                 lon=lon,
+                text=text,
                 mode="markers",
                 marker=dict(
                     size=15,
-                    color="#5C050B",
+                    color="#FFFF00",
```

Comment:
![image](https://github.com/bcdev/doors-dashboards/assets/144326622/e917cc9c-e9f3-4307-8e19-0cc89e773c82)
 the text is not visible in this color, so may be prefer a dark color.